doc/manual/trmacros.txt

```diff
@@ -43,6 +43,9 @@ Fortunately Nim supports side effect analysis:
   
   echo f() * 2 # not optimized ;-)
 
+You can make one overload for constrained match and without, and the constrained
```

Comment:
Something that has a constraint is constrained, but that's english, and ofc. it might feel a bit off to use eng. grammars on what is mostly viewed as conceptual words. I'll re-phrase it to use constraint.